tools/filamesh/src/main.cpp

```diff
@@ -222,6 +223,8 @@ static void printUsage(const char* name) {
                     "       Print copyright and license information\n\n"
                     "   --interleaved, -i\n"
                     "       interleaves mesh attributes\n\n"
+                    "   --meshopt, -m\n"
```

Comment:
Should we just call this flag --compress?

Reply:
Well, I was going to add a `--draco` flag next...

Reply:
--compress=method?

Otherwise we'll be able to set two incompatible flags. Also do we want Draco in filamesh? It'll make the filameshio library bigger. We should check how big Draco is after linking and stripping.

Reply:
Good idea, changed to --compress.